tour_model_eval/get_users.py

```diff
@@ -0,0 +1,32 @@
+import data_preprocessing as preprocess
+
+
+# to determine if the user is valid:
+# valid user should have >= 10 trips for further analysis and the proportion of filter_trips is >=50%
+def valid_user(filter_trips,trips):
+    valid = False
+    if len(filter_trips) >= 10 and len(filter_trips) / len(trips) >= 0.5:
+        valid = True
+    return valid
+
+
+# - user_ls: a list of all users, like [user1, user2, user3...]
+# - valid_user_ls: a list of valid users
+# - all_users: the collection of all user id
+def get_user_ls(all_users,radius):
+    user_ls = []
+    valid_user_ls = []
+    for i in range(len(all_users)):
+        curr_user = 'user' + str(i + 1)
+        user = all_users[i]
+        filter_trips,sim,trips = preprocess.filter_data(user,radius)
```

Comment:
Don't you need to remove this at a minimum? `filter_data` no longer returns 3 values.

```suggestion
        filter_trips,trips = preprocess.filter_data(user,radius)
```